coordinator/pom.xml

```diff
@@ -174,6 +174,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <version>3.7.1</version>
+        <configuration>
+          <descriptorRefs>
+            <descriptorRef>jar-with-dependencies</descriptorRef>
```

Comment:
I am not super sure if this will deploy to Maven central but it might do

Reply:
The tool is a standalone java program.